inc/saihostif.h

```diff
@@ -452,16 +455,20 @@ typedef enum _sai_hostif_user_defined_trap_type_t
      */
     SAI_HOSTIF_USER_DEFINED_TRAP_TYPE_START = 0x00000000,
 
-    /** Router traps (default packet action is drop) */
+    /** Router traps */
     SAI_HOSTIF_USER_DEFINED_TRAP_TYPE_ROUTER = SAI_HOSTIF_USER_DEFINED_TRAP_TYPE_START,
 
-    /** Neighbor table traps (default packet action is drop) */
+    /**
+     * @brief Neighbor table traps
+     *
+     * Generated by neighbor table entry hit with action trap/log, or by neighbor table miss
+     */
     SAI_HOSTIF_USER_DEFINED_TRAP_TYPE_NEIGH,
```

Comment:
Can we also change this to SAI_HOSTIF_USER_DEFINED_TRAP_TYPE_NEIGHBOR instad of having short name?